File: 2nd_june_2021/occ_of_digits.cpp
/*
to count Occurence of digit in the number
Date: 03/06/2021
*/
#include<iostream>
using namespace std;
void occ_of_digit(long long int number,int digit){
    int occ=0,rem;
    while(number){
        rem=number%10;
        if(rem==digit)
            occ++;
        number/=10;
    }
    cout<<"occ of "<<digit<<" in the number: "<<occ<<endl;
}

int main(){
    long long int number;
    int digit;
    cout<<"Enter the number: ";
    cin>>number;
    for(int i=0;i<=9;i++){
        occ_of_digit(number,i);
    }
    return 0;
}

File: PreviousCodesInJS/occ_of_digits.js
function occ_of_digit(number, digit) {
    var occ = 0,
        rem;
    while (number) {
        rem = number % 10;
        if (rem == digit)
            occ++;
        number = parseInt(number / 10);
    }
    console.log("occ of " + digit + " in the number: " + occ);
}

function occurrence(number) {
    for (var i = 0; i <= 9; i++) {
        occ_of_digit(number, i);
    }
}

occurrence(123456784534);